Tests/AppTests/WebpageSnapshotTests.swift

```diff
@@ -24,6 +24,7 @@ let recordSnapshotForAllTests = false
 class WebpageSnapshotTests: XCTestCase {
     
     override func setUpWithError() throws {
+        try XCTSkipIf((Environment.get("SKIP_SNAPSHOTS") ?? "false") == "true")
```

Comment:
I've added this so I can easily run the tests locally without the snapshots polluting the results. However, unless you run the tests every once in a while on Catalina we've got a bit of a test gap here. I'll add an issue to see if we can also run the tests on macOS in CI.

Reply:
#711

Reply:
We might get a release date for Big Sur tomorrow, maybe. I'll upgrade as soon as it hits GM.

Reply:
I just ran the tests on Catalina with snapshots on this branch and they pass 👍 I'll keep doing that before merging anything until we have this sorted out.